code/game/machinery/engi_points_vendor

```diff
@@ -0,0 +1,35 @@
+/*	
+	var/static/list/prize_list = list(
+		new /datum/GBP_equipment("Tendie",				/obj/item/weapon/reagent_containers/food/snacks/nugget,								50,		1),
+		new /datum/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,													50,		1),
+		new /datum/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,																							50,		1),
+		new /datum/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,																				250,	1),
+		new /datum/GBP_equipment("Advanced Indoor Fulton Pack",			/obj/item/weapon/extraction_pack/advanced,					300,	1),
```

Comment:
i disagree on the indoors fultons, it was good enough for peace walker

also its good for if you need to move a lot of resources fast